plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/task/MetalavaSignature.kt

```diff
@@ -7,6 +7,22 @@ import org.gradle.api.tasks.JavaExec
 import org.gradle.api.tasks.TaskProvider
 import java.io.File
 
+private val sourceLanguageDirectoryNames = listOf("java", "kotlin")
+
+// Predicate to determine whether a given directory contains exported APIs.
+private val isProductionCodeDirectory: (String) -> Boolean = { dirName ->
+    var isProductionCodeDirectory = true
+    run loop@{
+        listOf("test").forEach {
```

Comment:
Should this list be saved as a source of truth somewhere? Could it ever change?

Reply:
I'd just adhered to what was already there, really, and not considered allowing configuration. What I can do is create a a new extension property called, say, `excludeSourcePaths`, defaulting to be a list of the default android test dir names (`androidTest`, `test` etc), and those are to be considered as sub-directories of the directories provided by the `sourcePaths` property?

Reply:
Rethought this, and the PR update represents that change.